assets/js/components/surveys/CurrentSurvey.test.js

```diff
@@ -80,6 +81,122 @@ describe( 'CurrentSurvey', () => {
 		expect( container ).toMatchSnapshot();
 	} );
 
+	it( "should render an open text question when the `question_type` is 'open_text'", async () => {
```

Comment:
This test covers all the functionality of `SurveyQuestionOpenText`, and also its integration with `CurrentSurvey`

Reply:
This looks good in terms of testing functionality, but this is a very long test with many `expect` statements that seem unrelated.

Let's put everything related to `SurveyQuestionOpenText` in its own `describe` block, then separate out some of these `expect` statements.

Reply:
Sounds good @tofumatt 

I don't think I really got the mandate from Evan to do so many tests (see comment https://github.com/google/site-kit-wp/issues/3762#issuecomment-892736060) but I :heart: comprehensive testing as I :heart:  TDD

Reply:
I've broken up this test into smaller tests. Happy to make any more changes of course